distribution/server-x-dist/pom.xml

```diff
@@ -64,18 +64,6 @@
                                     <type>zip</type>
                                     <outputDirectory>target/</outputDirectory>
                                 </artifactItem>
-                                <artifactItem>
-                                    <groupId>org.keycloak</groupId>
-                                    <artifactId>keycloak-quarkus-server</artifactId>
-                                    <type>jar</type>
-                                    <outputDirectory>target/keycloak-quarkus-server</outputDirectory>
-                                </artifactItem>
-                                <artifactItem>
-                                    <groupId>org.keycloak</groupId>
-                                    <artifactId>keycloak-quarkus-server-app</artifactId>
-                                    <type>jar</type>
-                                    <outputDirectory>target/keycloak-quarkus-server-app</outputDirectory>
-                                </artifactItem>
```

Comment:
I don't understand how these changes are related to themes?

Reply:
Not related, but removing unnecessary code. Going to send a separate PR.